src/directives/modal.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileoverview Provides the "ngeoModal" directive. This directive allows
+ * displaying a modal window. The directive takes care or the show/hide
+ * actions.
+ *
+ *
+ * Example:
+ *
+ * <modal shown="modalShown">
+ *   <div class="modal-header">
+ *     <button type="button" class="close" data-dismiss="modal"
+ *         aria-hidden="true">&times;</button>
+ *     <h4 class="modal-title">The Title</h4>
+ *   </div>
+ *   <div class="modal-body">Some content</div>
+ * </modal>
+ *
+ * This directive requires Bootstrap's `modal` classes and associated jQuery
+ * plugin.
+ */
+
+goog.provide('ngeo.modalDirective');
+
+goog.require('ngeo');
+
+
+/**
+ * @param {angular.$parse} $parse Angular parse service.
+ * @return {angular.Directive} The directive specs.
+ */
+ngeo.modalDirective = function($parse) {
+  return {
+    template: '<div class="modal fade" tabindex="-1" role="dialog">' +
+        '<div class="modal-dialog">' +
+        '<div class="modal-content">' +
+        '<ng-transclude></ng-transclude>' +
+        '</div>' +
+        '</div>' +
+        '</div>',
+    restrict: 'E',
+    transclude: true,
+    replace: true,
+    link: function(scope, element, attrs) {
```

Comment:
I think it's a good idea to type the function. See https://github.com/camptocamp/ngeo/blob/master/src/directives/btngroup.js#L107-L113. In this way, `scope`, `element` and `attrs` are typed, and you're then guaranteed to 
correctly use these variables.